portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,110 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font.css">
+
     <script src="script.js"></script>
```

Comment:
Now that you've deleted the file, you'll need to delete this reference too. (See JS console for the error.)